lib/types/type_aliases.ex

```diff
@@ -0,0 +1,26 @@
+defmodule TypeAliases do
+  @moduledoc """
+  Type aliases for ssz schemas
+  """
+
+  def root, do: {:bytes, 32}
+  def epoch, do: {:int, 64}
+  def bls_signature, do: {:bytes, 96}
+  def slot, do: {:int, 64}
+  def commitee_index, do: {:int, 64}
+  def validator_index, do: {:int, 64}
+  def gwei, do: {:int, 64}
+  def participation_flags, do: {:int, 8}
+  def withdrawal_index, do: {:int, 64}
+  def bls_pubkey, do: {:bytes, 48}
+  def execution_address, do: {:bytes, 20}
+  def version, do: {:bytes, 4}
+  def domain, do: {:bytes, 32}
+  def bytes32, do: {:bytes, 32}
+  def uint64, do: {:int, 64}
+  def hash32, do: {:bytes, 32}
+  def uint256, do: {:int, 256}
+  def transaction, do: {:list, {:bytes, 8}, ChainSpec.get("MAX_BYTES_PER_TRANSACTION")}
```

Comment:
This is not a list of `{bytes, 8}` (which is 8 bytes) this is a list of bytes. For bytelists we may just use `List[uint8]` for now. That would be `{:list, {:int, 8}, size}`. If it's useful, we can add a type alias called

```elixir
def byte_list(n), do: {:list, {:int, 8}, n}
```